src/index.ts

```diff
@@ -3,6 +3,7 @@ type LoggingLevelType = 'debug' | 'info' | 'warn' | 'error' | 'fatal'
 
 interface LogifyPropsType<ErrorGenericType> {
   endpoint: string
+  storageKey?: string
```

Comment:
sorry, it seems that my previous comment was not clear enough: the key property can stay inside storage, i just meant using getItem()/setItem() over get()/set()!

Reply:
I'm not sure there would be a solution for the default value of storage key if the key is within the storage object. This way I would have to set the default value for the whole storage object and the if (!storage) { return } checks would never be executed. 
So, I moved the storageKey outside the storage object.

Reply:
the default doesn't need to be declared with the `storage`, it can come from somewhere else. for example, you can simply have a `const DEFAULT_STORAGE_KEY = 'logs'` and use it whenever you want to access the `key` as such for ie: `key ?? DEFAULT_STORAGE_KEY`. that way we still have `key?: string` and we also have the default value.

Reply:
✅